models/staging/stg_ga4_export__event.sql

```diff
@@ -70,17 +70,9 @@ final_pre as (
 
     from fields
 
-),
-
-exclude_intraday as (
-
-    select
-        *
-    from final_pre
-    where is_intraday = false
-
 )
 
 select
     *
-from exclude_intraday
+from final
+where not is_intraday
```

Comment:
Should we coalesce here to `true`. This way the null records are shown as true and therefore not included for sure downstream

Reply:
Thats a good point to account for nulls. But I think if `is_intraday` is null, that would mean false and those records should be included. I can ask the engineers to confirm whether it makes sense to coalesce to `false`

Reply:
When I coalesced to `false` I saw the uniqueness test for this model fail. I don't believe we should do a coalesce here to false.